project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.8
+sbt.version=0.13.12
```

Comment:
we probably shouldn't change sbt, scala, and sbt-native-packager versions in the middle of this PR, no?